ufl/constantvalue.py

```diff
@@ -12,6 +12,7 @@
 
 from math import atan2
 
+import ufl
```

Comment:
Circular?

Reply:
This seems fine so far, and it has been done to avoid Importing directly BaseForm which leads to a circular import. Another alternative would be to import `UFLType` and instead of checking if the expression passed in `as_ufl` is an `Expr` or a `BaseForm`, we could directly check if the type of this expression is `UFLType` since both classes have `UFLType` as metaclass. This wouldn't cause a circular import since we already import the `ufl_type` decorator from `ufl.core.ufl_type`